pkg/testhelpers/mirror.go

```diff
@@ -220,6 +227,33 @@ func AddOutputPortToMirror(portUUID, mirrorName string) (string, error) {
 	return strings.TrimSpace(string(output[:])), nil
 }
 
+// CreateEmptyMirrors creates multiple mirrors in a bridge with an optional owner in external_ids.
+// If ovsPortOwner is an empty string, it will leave external_ids empty.
+func CreateEmptyMirrors(bridgeName string, mirrorNames []string, ovsPortOwner string) {
+	for _, mirrorName := range mirrorNames {
+		_, err := createEmptyMirror(bridgeName, mirrorName)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		if ovsPortOwner != "" {
+			// manually add owner as external_ids
+			_, err = addOwnerToMirror(ovsPortOwner, mirrorName)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}
+		// check existence of the empty mirrors
+		emptyMirExists, err := IsMirrorExists(mirrorName)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(emptyMirExists).To(gomega.Equal(true))
```

Comment:
If createEmptyMirror executed without an error, I think it's safe to assume it is there. This code can be removed.

Reply:
In theory yes. I added this to be really sure about the state of mirror table.
If you want I can push a commit to remove it.

Reply:
If you would find spare moment to push a PR with the cleanup, that'd be nice, your call